src/site/content/en/fast/why-speed-matters/index.md

```diff
@@ -37,4 +38,96 @@ When it comes to user experience, speed matters. A [consumer study](https://www.
+On slower connections, however, users are forced to wait.
+Users may experience more problems as page resources slowly trickle in.
+
+<figure class="w-figure">
+  <img src="speed-comparison.png" alt="A comparison of two filmstrip reels
+of a page loading. The first shows a page loading on a slow connection, while
+the second shows the same page loading on a fast connection.">
+  <figcaption>A comparison of page load on a very slow connection
+(top) versus a faster connection (bottom).</figcaption>
+</figure>
+
+Performance is a foundational aspect of good user experiences.
+When sites ship a lot of code, browsers must use megabytes of the user's data plan in order to download the code.
+Mobile devices have limited CPU power and memory.
+They often get overwhelmed with what we might consider a small amount of unoptimized code.
+This creates poor performance which leads to unresponsiveness.
+Knowing what we know about human behavior, users will only tolerate low performing applications for so long before abandoning them.
+
+## Performance is about people
+
+Poorly performing sites and applications can also pose real costs for the
+people who use them.
+
+[As mobile users continue to make up a larger portion of internet users
+worldwide](http://gs.statcounter.com/platform-market-share/desktop-mobile-tablet),
+it's important to bear in mind that many of these users access the web through
+mobile LTE, 4G, 3G and even 2G networks.
+As Ben Schwarz of Calibre points out in
+[this study of real world performance](https://building.calibreapp.com/beyond-the-bubble-real-world-performance-9c991dcd5342),
+the cost of prepaid data plans is decreasing,
+which in turn is making access to the internet more affordable in places where it once wasn't.
+Mobile devices and internet access are no longer luxuries.
+They are common tools necessary to navigate and function in an increasingly interconnected world.
+
+[Total page size has been steadily increasing since at least 2011](http://beta.httparchive.org/reports/state-of-the-web#bytesTotal),
+and the trend appears to be continuing.
+As the typical page sends more data,
+users must replenish their metered data plans more often, which costs them money.
+
+In addition to saving your users money,
+fast and lightweight user experiences can also prove crucial for users in crisis.
+Public resources such as hospitals, clinics,
+and crisis centers have online resources that give users important and specific information that they need during a crisis.
+[While design is pivotal in presenting important information efficiently in stressful moments](https://aneventapart.com/news/post/eric-meyer-designing-for-crisis),
+the importance of delivering this information fast can't be understated.
+It's part of our job.
+
+## Test and improve your site
+
+The [Test My Site](https://www.thinkwithgoogle.com/feature/testmysite/)
+tool measure your site performance and gives recommendations for improvements.
```

Comment:
```suggestion
tool measures your site performance and gives recommendations for improvements.
```

Reply:
I'm going to do a quick rewrite of this section to focus on modern tools